.travis.yml

```diff
@@ -37,6 +38,7 @@ addons:
   firefox: latest
 
 before_install:
+  - mysql -e 'create database olympia;'
```

Comment:
This fails faster and we now have our database setup bundled in `before_install` step